package.json

```diff
@@ -21,7 +21,8 @@
 		"eslint-config-origami-component": "^1.0.0",
 		"expect": "^24.1.0",
 		"mocha": "^5.2.0",
-		"mock-require": "^3.0.3"
+		"mock-require": "^3.0.3",
+		"node": "^10.16.0"
```

Comment:
👀

Reply:
👀 that should probably be in a separate commit huh